aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -10,8 +10,19 @@
 #include <cstring>
 #include <cctype>
 
+#include <ATen/Dispatch.h>
+#include <ATen/native/DispatchStub.h>
+#include <ATen/native/TensorIterator.h>
+
 namespace at { namespace native {
 
+using unpack_pivots_fn = void(*)(
+  TensorIterator& iter,
+  int64_t dim_size
+);
+
+DECLARE_DISPATCH(unpack_pivots_fn, unpack_pivots_stub);
+
```

Comment:
It doesn't matter much but I think it's better to have this in the `LinearAlgebra.h` file.

Reply:
Looks like this could potentially solve the issue with mobile platforms. However, if I move this dispatch into `LinearAlgebra.h`, I get the following compilation error:
```
AILED: caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/torch_cuda_generated_LUUnpackKernel.cu.o 
cd /home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda && /home/nik/.conda/envs/pytorch-cuda-dev/bin/cmake -E make_directory /home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/. && /home/nik/.conda/envs/pytorch-cuda-dev/bin/cmake -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=/home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/./torch_cuda_generated_LUUnpackKernel.cu.o -D generated_cubin_file:STRING=/home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/./torch_cuda_generated_LUUnpackKernel.cu.o.cubin.txt -P /home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/torch_cuda_generated_LUUnpackKernel.cu.o.Release.cmake
/home/nik/git/pytorch/aten/src/ATen/native/cuda/LUUnpackKernel.cu(92): error: expected a type specifier

1 error detected in the compilation of "/home/nik/git/pytorch/aten/src/ATen/native/cuda/LUUnpackKernel.cu".
CMake Error at torch_cuda_generated_LUUnpackKernel.cu.o.Release.cmake:281 (message):
  Error generating file
  /home/nik/git/pytorch/build/caffe2/CMakeFiles/torch_cuda.dir/__/aten/src/ATen/native/cuda/./torch_cuda_generated_LUUnpackKernel.cu.o

```